ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -6,3 +6,9 @@ def det(input, name=None):
 
 
 det.unsupported_dtypes = ("float16", "bfloat16")
+
+
+def slogdet(input, name=None):
+    return ivy.slogdet(input)
+
```

Comment:
Please add a blank line here to fix the lint failure.

Reply:
done